### PR TITLE
Important notes (Dashboard Widget)

### PR DESCRIPTION
**Pull request by Schottkyc137**
see issue: #1238

### Visible/Frontend Changes
- [x] Add a dashboard widget that shows important notes
- [ ] Make the notes configurable to a user

### CODE REVIEW

File: src/app/child-dev-project/notes/dashboard-widgets/important-notes/important-notes.component.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          <td *matCellDef="let note" class="subject-cell">
+            {{ note.subject }}
+          </td>
+        </ng-container>
+
+        <tr
+          mat-row
+          *matRowDef="let row; columns: ['date', 'title'];"
+          class="dashboard-table-row"
+          (click)="openNote(row)"
```
Comment (TheSlimvReal) [inline src/app/child-dev-project/notes/dashboard-widgets/important-notes/important-notes.component.html:38]: Mouse should indicate that row is clickable (changing the cursor style)

**Review by TheSlimvReal — CHANGES_REQUESTED** [review]

**Review by sleidig — COMMENTED** [review]
Only review UI so far: I like how this looks and works overall :smiley: 

Are the notes sorted by anything? Maybe by date makes sense?
Possibly could also indicate somehow whether it's "Urgent" or only "Needs Followup"

**Comment by Schottkyc137** [comment]
All issues should be fixed now. I have also re-introduced the color as background since different notes can have a different color (red or orange)

**Review by TheSlimvReal — CHANGES_REQUESTED** [review]
Looks solid now @sleidig can you give a last UX review? I am still struggling a little with the very strong red color on all the warning notes.

**Comment by sleidig** [comment]
Very much like the widget overall. I do agree about the color being a bit overwhelming though. How about using the less heavy styling we have on the "record attendance" view, something like this:
image
(for reference, this is `.row-view { border-left: 8px solid orange; }`)

@TheSlimvReal I think you did the technical code review? I didn't look closer at that yet.

**Comment by Schottkyc137** [comment]
> Very much like the widget overall. I do agree about the color being a bit overwhelming though. How about using the less heavy styling we have on the "record attendance" view, something like this: image (for reference, this is `.row-view { border-left: 8px solid orange; }`. Doesn't look perfectly clean yet without spacing, maybe we should try with an extra, empty but colored td instead)
> 
> @TheSlimvReal I think you did the technical code review? I didn't look closer at that yet.

I very much like the idea of having a more subtle hint instead of changing the whole background color. What do you think about the following?
Bildschirmfoto 2022-08-30 um 18 22 26

Though this is not more than a potential alternative; I don't have any strong opinions on either design.

**Review by TheSlimvReal — APPROVED** [review]
Nice work, I like where we got with this and I think a lot of organisations can use this

**Pull request merged**